models/permission.js

```diff
@@ -0,0 +1,47 @@
+module.exports = (sequelize, DataTypes) => {
```

Comment:
Models names are always capitalized! Please consider correcting the name of this file.